backend/src/packages/chaiNNer_pytorch/pytorch/processing/upscale_image.py

```diff
@@ -107,6 +112,24 @@ def estimate():
                 hint=True,
             )
         ),
+        if_group(
+            Condition.type(1, "Image { channels: 4 } ")
+            & (
+                Condition.type(
+                    0, "PyTorchModel { inputChannels: 1, outputChannels: 1 }"
+                )
+                | Condition.type(
+                    0, "PyTorchModel { inputChannels: 3, outputChannels: 3 }"
+                )
+            )
+        )(
+            BoolInput("Separate Alpha", default=False).with_docs(
+                "Normally when dealing with an image with alpha, we take the difference between an"
+                " upscale with a black background and an upscale with a white background to get the"
+                " alpha channel. However, under certain circumstances it may be more desirable to"
+                " upscale the alpha channel separately from the RGB channels."
```

Comment:
I think this description is too technical. If I didn't know how these techniques worked already, I would have no idea what this description was trying to communicate.

How about we describe the properties of the 2 techniques instead? Basically, we describe their tradeoffs and why you would use one over the other.

> Upscale alpha separately from color. Enabling this option will cause the alpha of the upscaled image to be less noisy and more accurate to the alpha of the original image, but the image may suffer from dark borders near transparency edges (transition from fully transparent to fully opaque).
> Whether enabling this option will improve the upscaled image depends on the original image. We generally recommend this option for images with smooth transitions between transparent and opaque regions.